Snippets/Operations/Operations.csproj

```diff
@@ -109,6 +109,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
     <None Include="packages.config" />
+    <None Include="SqlServer\NativeSendPowershell.ps1" />
```

Comment:
why both `NativeClient.ps1` and `NativeSendPowershell.ps1`? is one of them redundant? and `NativeClient.ps1` is not in the sln